package.json.njk

```diff
@@ -39,7 +39,6 @@
     "openapi-nodegen-logger": "^1.0.0",
     "recursive-readdir-sync": "^1.0.6",
     "request-ip": "^2.1.3",
-    "tslib": "^1.10.0",
```

Comment:
No way, i don't know how 2 tslibs got in there. However, the server will require the tslib in the dependencies not the dev deps.

Reply:
I'm not going to lie - I may have tried to install it a few times before I realized what was going on...

I'll switch it - I just assumed as a ts package it was a devDep, I didn't look into what it does.